.github/workflows/ci_test-full.yml

```diff
@@ -122,6 +122,7 @@ jobs:
         pip --version
         # python -m pip install --upgrade --user pip
         flag=$(python -c "print('--pre' if '${{matrix.release}}' == 'pre' else '')" 2>&1)
+        pip install git+https://github.com/PyTorchLightning/metrics.git@apply_sync_fn
```

Comment:
```suggestion
        # fixme: temove after merging to master
        pip install git+https://github.com/PyTorchLightning/metrics.git@apply_sync_fn
```

Reply:
This will be removed on merge. Used for CI testing purpose.

Reply:
Done !